tests/acceptance/YotiAdminCest.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+class YotiAdminCest
```

Comment:
Is this meant to be 'Cest' or 'Test'?

Reply:
It's a Codeception thing: https://codeception.com/docs/07-AdvancedUsage#Cest-Classes